pkg/process/config/yaml_config.go

```diff
@@ -157,12 +157,10 @@ func (a *AgentConfig) loadProcessYamlConfig(path string) error {
 		//   If "true" we will collect containers and processes.
 		//   If "disabled" the agent will be disabled altogether and won't start.
 		enabled := config.Datadog.GetString(k)
-		if ok, err := isAffirmative(enabled); ok {
+		if ok, _ := isAffirmative(enabled); ok {
 			a.Enabled, a.EnabledChecks = true, processChecks
 		} else if enabled == "disabled" {
 			a.Enabled = false
-		} else if !ok && err == nil {
```

Comment:
This doesn't look right? `a.Enabled` would be false then?

Reply:
It will default to the default value (it's `true` if it detected containers, false otherwise)